content/blog/2016-09-14-interview-with-digitalocean.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Interview with DigitalOcean
+created_at: 2016-09-14
+kind: article
+author_name: Brian Brazil
+---
+
+*Next in our series of interviews with users of Prometheus, DigitalOcean talks
+about how they use Promethus. Carlos Amedee also talked about [the social
+aspects of the rollout](https://www.youtube.com/watch?v=ieo3lGBHcy8) at PromCon
```

Comment:
I meant linking both, but we could also change all links to promcon.io which prominently displays the slides to each talk.